pkg/oras/store.go

```diff
@@ -65,3 +67,54 @@ func (s *hybridStore) Writer(ctx context.Context, opts ...content.WriterOpt) (co
 	}
 	return s.ingester.Writer(ctx, opts...)
 }
+
+// TODO: implement (needed to create a content.Store)
+// TODO: do not return empty content.Info
+// Abort completely cancels the ingest operation targeted by ref.
+func (s *hybridStore) Info(ctx context.Context, dgst digest.Digest) (content.Info, error) {
+	return content.Info{}, nil
+}
+
+// TODO: implement (needed to create a content.Store)
+// Update updates mutable information related to content.
+// If one or more fieldpaths are provided, only those
+// fields will be updated.
+// Mutable fields:
+//  labels.*
+func (s *hybridStore) Update(ctx context.Context, info content.Info, fieldpaths ...string) (content.Info, error) {
+	panic("not yet implemented: Update (content.Store interface)")
```

Comment:
Should we return `error` instead of `panic()`?

Reply:
done